### PR TITLE
Update Dataprep with Parameter Settings

### PR DESCRIPTION
**Pull request by letonghan**
## Description

Update Dataprep with Parameter Settings

## Issues

1. https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L132C5-L138C68 (embedding model -> docker_compose.yaml)
2. https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L120 (chunk_overlap)
3. https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L38 (redis host)

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

None

## Tests

Local test and CI test

### CODE REVIEW

File: comps/dataprep/redis/README.md [hunk 1]
```diff
@@ -69,6 +95,10 @@ Please refer to this [readme](../../../vectorstores/langchain/redis/README.md).
 ## 2.2 Setup Environment Variables
 
 ```bash
+export EMBEDDING_MODEL_ID="BAAI/bge-base-en-v1.5"
+export TEI_ENDPOINT="http://${your_ip}:6006"
```
Comment (lkk12014402) [inline comps/dataprep/redis/README.md:99]: we should make modifications for GenAIExamples, like https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/compose.yaml#L23

File: comps/dataprep/redis/langchain/docker/docker-compose-dataprep-redis.yaml [hunk 2]
```diff
@@ -21,6 +34,8 @@ services:
       no_proxy: ${no_proxy}
       http_proxy: ${http_proxy}
       https_proxy: ${https_proxy}
+      REDIS_HOST: ${REDIS_HOST}
```
Comment (lkk12014402) [inline comps/dataprep/redis/langchain/docker/docker-compose-dataprep-redis.yaml:37]: the `TEI_ENDPOINT` is not set here？we need this env in the code https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L36

File: comps/dataprep/redis/README.md (hunk 1 above)
Comment (letonghan) [reply comps/dataprep/redis/README.md:99]: you mean export `6006` port number as an environment variable?

Comment (lkk12014402) [reply comps/dataprep/redis/README.md:99]: I mean the docker compose files of chatqna example should export/import the `TEI_ENDPOINT` at the line https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/compose.yaml#L14? so you need push a pr to modify the GenAIExample?

**Review by lvliang-intel — APPROVED** [review]

**Review by lkk12014402 — APPROVED** [review]

**Pull request merged**